css/word-wrap/demos/basic/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Пример стилизации первой буквы в тексте — ::first-letter — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    *, *::before, *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    body {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      min-height: 100vh;
+      margin: 0;
+      padding: 50px;
+      background-color: #18191c;
+      color: #fff;
+      font-family: "Roboto", sans-serif;
+      font-size: 16px;
+      letter-spacing: 0.04rem;
+    }
+
+    p {
+      border: 1px solid #fff;
+      width: 100px;
+      word-wrap: break-word;
+    }
```

Comment:
```suggestion
    body {
      display: flex;
      align-items: center;
      justify-content: center;
      min-height: 100vh;
      margin: 0;
      padding: 50px;
      background-color: #18191C;
      color: #FFFFFF;
      font-family: "Roboto", sans-serif;
      font-size: 18px;
      letter-spacing: 0.04rem;
    }

    p {
      width: 130px;
      background-color: #2E9AFF;
      color: #000000;      
      word-wrap: break-word;
    }
```

Привет и с новым годом! :)

Небольшое предложение сделать демку немного крупнее и причесать под общий стилёк. Перенос по символам сохраняется :)